pwndbg/dbg/lldb/__init__.py

```diff
@@ -1089,15 +1105,25 @@ def send_remote(self, packet: str) -> bytes:
         response = self.dbg._execute_lldb_command(f"process plugin packet send {packet}")
 
         # We need extract response
-        for line in response.split("\n"):
-            if line.startswith("response: "):
-                ret = line[10:]
-                if not ret:
-                    continue
-                return ret.encode()
+        try:
+            idx = response.index("\nresponse: ")
+        except ValueError:
+            # Packets not implemented return empty
+            return b""
+
+        try:
+            is_err = response.index("\nresponse: \nerror: ") > -1
```

Comment:
We already have an index of `"\nresponse: "`. Wouldn't it be better to do such a test instead?:

```suggestion
            is_err = response[idx:].startswith("\nerror: ")
```